common/util/autoware_global_parameter_loader/Readme.md

```diff
@@ -0,0 +1,18 @@
+# Autoware Global Parameter Loader
+
+This package is to get global parameters.
```

Comment:
get global parameters

set (global) ROS parameters to each node?

Reply:
de06326